fern/docs/pages/developer_resources/environments.mdx

```diff
@@ -5,26 +5,50 @@ slug: developer_resources/environments
 ---
 
 
-Schematic supports multiple environments (e.g. staging and production) to support your development workflow.
+Schematic supports multiple environments (e.g. development and production) to support your development workflow.
 
 Resources in Schematic are predominately environment-scoped, but some are available account-wide (regardless of the selected environment). The table below highlights available resources and how they are scoped.
 
-| Resource type | Account/environment scope | Originates from | Concept exists in | Definition / Notes |
+
+
+## Account Scoped Resources
+
+Account scoped resources are the same across all environments. For example, a plan is scoped to your entire Account, so whenever you make a plan, it is accessible across all environments. This is helpful to create new plans in development and then they'll be ready in production without additional release work.
+
+| Resource type | Definition / Notes |
+| --- | --- |
+| Environment | An environment (e.g. development, staging, production) |
+| Plan | A plan is a collection of features that can be assigned to a company |
+| Feature | A feature is a product or service that can be enabled or disabled for a company |
+| Feature flag | A feature flag is a way to enable or disable a feature for a company |
+
+## Environment Scoped Resources
```

Comment:
It may make sense to just leave this section out, since most resources are environment-scoped. Less to maintain/keep up to date. We have content elsewhere that defines what resources are.

Reply:
I see your point -- just say "this set of things (account based resources)" work a bit differently than you may expect